### PR TITLE
Feat : added wishlist property

### PR DESCRIPTION
**Pull request by Mohasin-Haque**
I have added wishlist functionality. 
> Add to wishlist
> Remove to wishlist

### CODE REVIEW

File: src/components/cardsComp/cartCard/CartCard.jsx [hunk 1]
```diff
@@ -39,13 +37,11 @@ const CartCard = (product) => {
                                 </div>
                             </div>
                             <div className="action-container">
-                                <button className="card-btn action secondary-action font-weight" to="#">Move To Wishlist</button>
+                                <button className="card-btn action secondary-action font-weight" to="#" >Move To Wishlist</button>
```
Comment (syed-shaida-hussain) [inline src/components/cardsComp/cartCard/CartCard.jsx:40]: How are you moving to wishlist ? There is no click handler

**Comment by Mohasin-Haque** [comment]
recording


<details><summary>Browser metadata</summary>





```
Path:      /cart
Browser:   Chrome 100.0.4896.88 on Windows 10
Viewport:  1366 x 617 @1x
Language:  en-GB
Cookies:   Enabled
```

[Open in BrowserStack](https://www.browserstack.com/user/try-live?url=https%3A%2F%2Fdeploy-preview-6--fast-cart-react.netlify.app%2Fcart&os=Windows&os_version=10&browser=Chrome&browser_version=100.0&resolution=1366x617&speed=1&start=true&ref=netlify-source)



</details>

[Open Deploy Preview](https://deploy-preview-6--fast-cart-react.netlify.app/cart) · [Mark as Resolved](https://app.netlify.com/cdp/resolve?deployID=625beb394a0ab60008e71f2a&commentID=625bed24196b7745a5bcea46&resolution=resolved)

**Review by ajit-sharmaGH — APPROVED** [review]
Good naming convention 
you can merge it
nice UI

**Review by syed-shaida-hussain — APPROVED** [review]
There is no click handler on move to wishlist button . How are you moving items from cart to wishlist ? Everything else  looks clean , you can merge it .

**Pull request merged**